main.py

```diff
@@ -12,15 +12,15 @@
 S3_BUCKET = os.environ.get('S3_BUCKET')
```

Comment:
The comment above this line says that this is a `Cloudformation` configuration setting. Isn't this an environment variable setting regardless of using CloudFormation or not? If so, let's remove the word `CloudFormation` to avoid confusion.

Reply:
Done.